Libraries/LibIPC/Encoder.cpp

```diff
@@ -141,6 +142,13 @@ Encoder& Encoder::operator<<(const String& value)
     return *this << value.view();
 }
 
+Encoder& Encoder::operator<<(const ByteBuffer& value)
+{
+    *this << static_cast<i32>(value.size());
+    m_buffer.append(value.data(), value.size());
+    return *this;
+}
```

Comment:
I briefly thought about using a `SharedBuffer` for this here, but I decided not to involve another layer of abstraction.
I could probably do that if we don't want to pass potentially big buffers around (kinda weird as we support passing them as `StringView`s anyway)